### PR TITLE
Feature/project upload

### PR DESCRIPTION
**Pull request by sirolad**

### CODE REVIEW

File: app/Http/Controllers/ProjectController.php [hunk 1]
```diff
@@ -2,20 +2,26 @@
 
 namespace Pibbble\Http\Controllers;
 
+use Auth;
 use Pibbble\Project;
 use Illuminate\Http\Request;
 use Pibbble\Http\Requests;
 use Pibbble\Http\Controllers\Controller;
+use Cloudinary\Uploader;
```
Comment (unicodeveloper) [inline app/Http/Controllers/ProjectController.php:10]: Please arrange this from shortest to longest. I have said this before.

Comment (sirolad) [inline app/Http/Controllers/ProjectController.php:10]: Rearranged.

Best Regards,

_Surajudeen Akande_
Software Developer, _Andela_
+2348037690966 | Skype: sirolad https://webapp.wisestamp.com/#
http://facebook.com/runthinz  http://twitter.com/babatoclean1
http://github.com/andela-sakande

Comment (unicodeveloper) [inline app/Http/Controllers/ProjectController.php:10]: Where's the PR for that?

Comment (sirolad) [inline app/Http/Controllers/ProjectController.php:10]: I am sending it right away.

Best Regards,

_Surajudeen Akande_
Software Developer, _Andela_
+2348037690966 | Skype: sirolad https://webapp.wisestamp.com/#
http://facebook.com/runthinz  http://twitter.com/babatoclean1
http://github.com/andela-sakande

On Mon, Nov 9, 2015 at 11:16 AM, Prosper Otemuyiwa <notifications@github.com

**Pull request merged**